entrypoint/secretsencrypt/encryption_suite_test.go

```diff
@@ -56,17 +55,6 @@ var _ = BeforeSuite(func() {
 		Expect(os.Getenv("server_flags")).To(ContainSubstring("secrets-encryption:"),
 			"ERROR: Add secrets-encryption:true to server_flags for this test")
```

Comment:
Should exit out using `os.Exit(1)` as the test will continue to run even if this fails

Reply:
Fixed.